code/game/objects/items/mantis.dm

```diff
@@ -0,0 +1,41 @@
+/obj/item/mantis/blade
+	name = "mantis blade"
+	desc = "Powerful inbuilt blade, hidden just beneath the skin. Singular brain signals directly link to this bad boy, allowing it to spring into action in just seconds."
```

Comment:
Grammar fix.

"A blade designed to be hidden just beneath the skin. The brain is directly linked to this bad boy, allowing it to spring into action."